server/rooms.ts

```diff
@@ -35,7 +35,7 @@ import {PM as RoomBattlePM, RoomBattle, RoomBattlePlayer, RoomBattleTimer} from
 import {RoomGame, RoomGamePlayer} from './room-game';
 import {Roomlogs} from './roomlogs';
 import * as crypto from 'crypto';
-import {RoomAuth} from './user-groups';
+import {RoomAuth, PLAYER_SYMBOL} from './user-groups';
```

Comment:
```suggestion
import {RoomAuth} from './user-groups';
```

Reply:
tbh should we even be importing `RoomAuth` at all?

Reply:
It's used in the `BasicRoom` constructor to provide the auth handler:
```ts
this.auth = new RoomAuth(this);
```

Reply:
Hm, this should plausibly be `new Users.RoomAuth` (which should be exported from Users if it isn't currently).